src/uct/api/v2/uct_v2.h

```diff
@@ -775,6 +775,31 @@ typedef struct {
 } uct_md_attr_v2_t;
 
 
+/**
+ * @ingroup UCT_MD
+ * @brief  Memory domain capability flags.
+ */
+enum {
+    UCT_MD_FLAG_V2_FIRST       = UCT_MD_FLAG_LAST,
+
+    /**
+     * MD supports invalidation of memory handle registered with
```

Comment:
maybe we should phrase it like

MD supports invalidation of remote keys packed by ... for put/get operations

since we could register the key with several rights, but invalidate only for atomics and/or rma access

Reply:
MD -> memory domain